Simplenote/src/main/java/com/automattic/simplenote/BottomSheetDialogBase.java

```diff
@@ -5,7 +5,6 @@
 import android.support.design.widget.BottomSheetDialog;
 import android.view.WindowManager;
 
-import com.automattic.simplenote.R;
```

Comment:
Removing unused imports is always a good practice.  Since this file has no other changes as part of this pull request, it's arguably better to add that to a separate pull request or commit.